### DISCUSSION TITLE
按照B站官方教程，运行时没有报错，但页面是空白的没有展示地图是什么原因呢？

### DISCUSSION THREAD

**Discussion by Jason-gyj** [antvis/L7#2054] [Q&A]
![微信截图_20231114220126](https://github.com/antvis/L7/assets/57826966/5919482d-7f98-412e-8cf6-a5c76c7d40dc)

**Comment by lzxue** [comment]
如使用高德地图需要申请token，https://github.com/antvis/L7/discussions/1112

**Reply by Jason-gyj** [reply]
怎么操作？

**Comment by lzxue** [comment]
https://codesandbox.io/s/l7-gao-de-demo-3zgxqf?file=/index.js
看上面的demo 代码示例。

L7 的token 对应的高德 key

高德 key 介绍： https://lbs.amap.com/api/javascript-api-v2/guide/abc/jscode

![image](https://github.com/antvis/L7/assets/4366658/71e8516e-c305-44bc-b1ff-591daa5d6984)

全局设置密钥

![image](https://github.com/antvis/L7/assets/4366658/bc3d9757-bd1b-4657-97cf-57bd68b339ab)
